### DISCUSSION TITLE
Notifications

### DISCUSSION THREAD

**Discussion by berkut1** [symfony/symfony#46933] [Q&A] [answered]
Hi, 
I'm looking to create a notification system similar to what's on github, facebook, etc.

I'm trying to understand this https://symfony.com/doc/current/notifier.html but it doesn't seem to fit for me.

Can someone correct me? If I want to create something similar to github or facebook notifications I need to create something similar to laravel database notifications https://laravel.com/docs/9.x/notifications#database-notifications (because database notifications seem to be what I'm looking for), but using Symfony's notifier interfaces, or does Symfony's notifier have similar ones out of the box?

Thanks.

**Comment by norkunas** [comment, marked as answer]
Hey,

There is not direct integration because Symfony does not have a database/orm on it's own.
If you are using doctrine orm you could check an example here: https://gist.github.com/norkunas/987c82e964027c4d31f37d4197fd2931

**Reply by berkut1** [reply]
Thanks for the example.
